lib/src/views/pokedex/PokedexScreen.dart

```diff
@@ -5,45 +5,70 @@ import 'package:flutter_stuff/src/views/pokedex/Pokemon.dart';
 import 'package:http/http.dart' as http;
 import '../../models/PokemonsModel.dart';
 
-class PokedexScreen extends StatelessWidget {
+final NUMBER_OF_ITEMS_TO_LOAD = 5;
+
+class PokedexScreen extends StatefulWidget {
+  @override
+  _PokedexScreenState createState() => new _PokedexScreenState();
+}
+
+class _PokedexScreenState extends State<PokedexScreen> {
+  int _index = 1;
+  List<PokemonsModel> pokemons = [];
+  ScrollController _scrollController = new ScrollController();
+
+  @override
+  void initState() {
+    super.initState();
+    fetchFive();
+
+    _scrollController.addListener(() {
+      if (_scrollController.position.pixels ==
+          _scrollController.position.maxScrollExtent) {
+        fetchFive();
+      }
+    });
+  }
```

Comment:
I think this will make it more readable 

```suggestion
    _scrollController.addListener(_fetcherListener);
  }
```